### PR TITLE
fix/BOAS-217-fix-npm-run-dev-in-Windows

### PR DESCRIPTION
**Pull request by DavidJWhiteDev**
## Describe your changes
npm run dev and git commit were both failing on a windows machine, giving a Lifecycle error.  It appears that npm run script commands in package.json that in turn call npm run commands have an in windows if there is a colon in the script name - eg prisma:generate in this case.  Changing to prisma-generate solves this issue, allowing run and commit to work.

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "test": "NODE_ENV=test cross-env ava --no-worker-threads",
     "coverage": "c8 --reporter=html --reporter=text ava",
-    "dev": "npm run prisma:generate && nodemon --experimental-json-modules -e js ./src/server.js",
+    "dev": "npm run build && nodemon --experimental-json-modules -e js ./src/server.js",
```
[the remark this answers is not in the document]
Comment (lutamKC) [reply apps/api/package.json:16]: can you see it now? @DavidJWhiteDev

**Review by francescobrenci-AR — CHANGES_REQUESTED** [review]

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**